### PR TITLE
Fix the ui displaying last year's instructors

### PR DESCRIPTION
**Pull request by SamZhang02**
this is also good on explore page now

### CODE REVIEW

File: client/src/components/CourseTerms.tsx [hunk 1]
```diff
@@ -98,69 +99,71 @@ export const CourseTerms = ({ course, variant, query }: CourseTermsProps) => {
...
-                    >
-                      <div className='flex items-center space-x-1.5 whitespace-nowrap'>
-                        <SeasonIcon term={term} variant={variant} />
-                        <div className='pr-1 font-medium dark:text-gray-200'>
-                          <Highlight
-                            text={ins.name}
-                            query={query || undefined}
-                          />
+      {instructorGroups
+        .sort((a, b) => compareTerms(a[0], b[0]))
```
Comment (SamZhang02) [inline client/src/components/CourseTerms.tsx:103]: @39bytes i literally just added 1 line of sort here github is tripping with its change viewers

**Review by 39bytes — APPROVED** [review]

File: client/src/lib/utils.ts [hunk 2]
```diff
@@ -21,17 +20,17 @@ export const groupCurrentCourseTermInstructors = (course: Course) => {
 
   const entries = Object.entries(termGroups);
 
-  const indexOfTerm = (term: string) =>
-    COURSE_TERM_ORDER.indexOf(term.split(' ')[0]);
-
-  return entries.sort(([a], [b]) => indexOfTerm(a) - indexOfTerm(b));
+  return entries;
```
Comment (39bytes) [inline client/src/lib/utils.ts:23]: maybe just return `termGroups` instead

File: client/src/components/CourseTerms.tsx [hunk 3]
```diff
@@ -98,69 +99,71 @@ export const CourseTerms = ({ course, variant, query }: CourseTermsProps) => {
...
-                      to={`/instructor/${encodeURIComponent(ins.name)}`}
-                    >
-                      <div className='flex items-center space-x-1.5 whitespace-nowrap'>
-                        <SeasonIcon term={term} variant={variant} />
-                        <div className='pr-1 font-medium dark:text-gray-200'>
-                          <Highlight
-                            text={ins.name}
-                            query={query || undefined}
-                          />
+      {instructorGroups
```
Comment (39bytes) [inline client/src/components/CourseTerms.tsx:102]: ...then do `Object.entries` here

**Comment by terror** [comment]
Looking Amazing

**Review by 39bytes — APPROVED** [review]

**Pull request merged**